docs/output.md

```diff
@@ -357,6 +359,29 @@ Output Summaries:
 
 [RGI](https://github.com/arpcard/rgi) (**R**esistance **G**ene **I**dentifier) predicts resistome(s) from protein or nucleotide data based on homology and SNP models. It uses reference data from the Comprehensive Antibiotic Resistance Database (CARD).
 
+#### argNorm
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `normalized/`
+  - `*.{tsv}`: search results in tabular format
+  </details>
+  <details markdown="1">
+  <summary>ARG summary table headers</summary>
+
+| Table column                 | Description                                                                      |
```

Comment:
It is actually collapsed: 
![image](https://github.com/user-attachments/assets/c34b342c-eb8c-4dbd-a915-8652e9eade70)
But it should go to the summary tools section like hamronization and co. since it does not predict ARGs itself.

Reply:
Especially since it takes hamronization output as input, I would classify argNorm as a summary tool. So, please add this paragraph to section below: `### Summary tools`